contribs/gmf/src/controllers/defaultConfig.js

```diff
@@ -0,0 +1,15 @@
+goog.provide('gmf.defaultConfig');
+
+gmf.defaultConfig = angular.module('gmfDefaultConfig', [
```

Comment:
Shouldn't we put gettext and tmh.dynamicLocale here?

Reply:
Yes, I forgot that.